### ISSUE TITLE
Report user to Tickets bot in mutual servers button

### ISSUE THREAD

**Issue by Arimodu** [Vencord/plugin-requests#1063]
### Plugin Description

Looking for a plugin that would: 
- Add a button somewhere in a DM that would look through mutual servers with that person for the Tickets bot
- If it finds it, it should look for a channel with a message from the tickets bot and if it finds that, jump to that channel. 
- If it doesnt find it, search for keywords like 'ticket', 'mod', 'modmail', 'contact', etc and list channels that match in a popup, and jump to the selection on click. 
That is the base functionality I am looking for. 

Now the following is extra features that I feel somewhat borderline enter selfbot territory so if it falls into the "not possible" category just let me know and ignore them.
- If tickets bot message with create ticket interaction is found, use configurable title for the ticket and automatically create it (with popup to confirm), then jump directly to the ticket.
- Send configurable canned message in ticket with configurable fields like {username}, {date created of account}, {mutual servers}, etc...
- Attach file with DM transcript from both sides, use message logger messages in case deleted.

Basically I am looking to automate as much of the report process as possible. I would try my hand at this myself but I cant find any getting started or similar about making a vencord plugin. 
I am willing to make this into a paid commission. I get a lot of spam DMs and just the base functionality would help tremendously.

### Request Agreement

- [X] I read the above and made sure my plugin is possible and allowed
- [X] I checked the [existing plugins](https://vencord.dev/plugins) and made sure my plugin doesn't already exist
- [X] I [searched the existing issues](https://github.com/Vencord/plugin-requests/issues?q=is%3Aissue) and made sure a similar request doesn't already exist

**Comment by MrDiamondDog** [comment]
This is so incredibly niche

**Comment by Arimodu** [comment]
Ye I expected that... Thats why A: I thought about making it myself and B: I offered to make it a commission for someone
Ofc at the end of the day I can keep doing this manually, but I thought I would give this a shot. If you think this has no way of someone making it just tell me and I'll close this

**Comment by Vendicated** [comment]
way too niche and honestly trying to get someone banned from all mutual servers is not really something we want to encourage
